components/script/dom/xmlhttprequest.rs

```diff
@@ -726,6 +729,32 @@ impl XMLHttpRequestMethods for XMLHttpRequest {
                         return NullValue();
                     }
                 }
+                XMLHttpRequestResponseType::Blob => {
+                    match self.response_blob.get() {
```

Comment:
This should really go into a separate method down there, named `blob_response`, with an annotation containing the corresponding link to the WHATWG specification.